packages/@expo/fingerprint/src/utils/Profile.ts

```diff
@@ -1,5 +1,7 @@
 import chalk from 'chalk';
 
+import { NormalizedOptions } from '..';
```

Comment:
```suggestion
import type { NormalizedOptions } from '../Fingerprint.types';
```

- lean toward for having explicit "import type"
- import from Fingerprint.types to prevent import cycle